packages/node/src/indexer/dictionary/substrateDictionary.service.ts

```diff
@@ -101,3 +104,26 @@ export class SubstrateDictionaryService extends DictionaryService<
     return dict.getSpecVersions();
   }
 }
+
+function convertGetDsProcessor(
```

Comment:
This is a functional change and should have a test.

I think we will end up implementing this over for each SDK which isn't ideal. 

Could this not be fixed by improvements to the dictionary types? E.g. doing something like `typeof DsProcessorService["getDsProcessor"]`